esss_fix_format/cli.py

```diff
@@ -157,6 +159,22 @@ def _process_file(filename, check, format_code):
         click.secho(click.format_filename(filename) + ': Unknown file type', fg='white')
         return changed, errors, formatter
 
+    if is_cpp(filename):
+        with io.open(filename, 'rb') as f:
+            content_bytes = f.read()
+            content = content_bytes.decode()
+            non_ascii = re.sub('[ -~]', '', content).strip()
```

Comment:
I will suggest to use escapes in the regex or at least a comment specifying that `' '` represents the first printable ASCII character and `'~'` the last one.

Reply:
Done